src/VisualStudio/Core/Def/Implementation/DesignerAttribute/VisualStudioDesignerAttributeService.cs

```diff
@@ -316,7 +317,10 @@ private void AddFilteredInfos(ImmutableArray<DesignerAttributeData> data, ArrayB
         public Task ReportDesignerAttributeDataAsync(ImmutableArray<DesignerAttributeData> data, CancellationToken cancellationToken)
         {
             Contract.ThrowIfNull(_workQueue);
-            _workQueue.AddWork(data);
+
+            using var _ = ArrayBuilder<DesignerAttributeData>.GetInstance(out var temp);
+            temp.AddRange(data);
+            _workQueue.AddWork(temp);
```

Comment:
>_workQueue.AddWork(temp); [](start = 12, length = 25)

I think we should keep the original code and overload.

Reply:
can do.